.release/ci.hcl

```diff
@@ -0,0 +1,210 @@
+schema = "1"
+
+project "vault" {
+  team = "vault"
+  slack {
+    notification_channel = "CRF6FFKEW" // #vault-releases
+  }
+  github {
+    organization = "hashicorp"
+    repository = "vault"
+    release_branches = ["release/1.9.x"]
```

Comment:
I noticed that below there's a `version.sh` script which seems to be used by grabbing the vault version from the sdk (I think). Can the targeting of the release branch be automated a similar way, depending on the sdk version?

Reply:
Unfortunately not really, the release branch specified here is for the ci system to know explicitly which branches it should perform post-build actions on. We can probably populate this value by using `version.sh` in the upcoming `init` command though, for setting up new branches.